app/models.js

```diff
@@ -8,4 +9,10 @@ exports.fetchCategories = () => {
        return result.rows;
       });
   };
-  
\ No newline at end of file
+
+  exports.fetchEndpoints = () => {
```

Comment:
As you're not querying the database you dont need anything in your model. This can go in your controller 🙂